pkg/blobinfocache/memory/memory_test.go

```diff
@@ -0,0 +1,16 @@
+package memory
+
+import (
+	"testing"
+
+	"github.com/containers/image/pkg/blobinfocache/internal/test"
+	"github.com/containers/image/types"
+)
+
+func newTest(t *testing.T) (types.BlobInfoCache, func(t *testing.T)) {
```

Comment:
Rename this to `newCache` or `newTestCache`, please (or keep the original `newTestMemoryCache` name).

Reply:
> Rename this to `newCache` or `newTestCache`

`newTestCache` with b2f8be9 -> d545a19.